calico/reference/resources/ipamconfig.md

```diff
@@ -0,0 +1,43 @@
+---
+title: IPAM configuration
+description: IP address management global configuration
+canonical_url: '/reference/resources/ipamconfig'
+---
+
+A IPAM configuration resource (`IPAMConfiguration`) represents IPAM specific global configuration options.
```

Comment:
English is not my first language so I could be way off here, but shouldn't it be "**An** IPAM configuration resource..."?

Reply:
Yep, definitely should be.